app/controllers/articles_controller.rb

```diff
@@ -12,22 +13,34 @@ def create
   end
 
   def show
-    @article = Article.find_by(id: params[:id])
+    @article
```

Comment:
There's no need for this line. The article is available in this action due to the `before_action`